### ISSUE TITLE
Files 3.7 feels much slower than 3.6

### ISSUE THREAD

**Issue by Steperman5692** [files-community/Files#16180]
### Description

I have notice that Files 3.7 feels much slower than 3.6.
I use Files as my daily driver and I have notice a performance decrease.

### Steps To Reproduce

Navigate folders in Files.
Compare Files 3.6 to 3.7

### Requirements

Fix the performance decrease in Files 3.7

### Files Version

3.7.0.0

### Windows Version

22631.4169

### Log File

[debug.log](https://github.com/user-attachments/files/16979043/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, Slower in what way, does it happen after you do something in Files?

**Comment by Steperman5692** [comment]
Like navigating folders and click them there like a 1 or 2 second delay.

**Comment by Josh65-2201** [comment]
Can you see if this is an issue in the preview version. https://files.community/download

**Comment by Steperman5692** [comment]
Nope, the load time is way quicker, clicking folders is fast and navigating Files is faster.

**Comment by Josh65-2201** [comment]
Great, There will likely be a update realest to sable soon with the fixes of preview
